packages/rancher-alerting-drivers/charts/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for rancher-alerting-driver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  cattle:
+    # the registry where all images will be pulled from
+    systemDefaultRegistry: ""
+  preInstallHook:
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 1000
+    image:
+      repository: rancher/rancher-agent
+      tag: v2.5.7 # TODO: change to v2.5.8
```

Comment:
Whats pre-install hook for ?

Reply:
To either create the configMaps if they do not exist or skip the creation if they exist. 

So the configMaps are not affected by updating/uninstalling the app, so user's configurations are preserved.

Reply:
Ok. Now question is how are we going to keep them up to date for airgap, ideally we don't come in and change this for every rancher release. Is this used elsewhere in monitoring ?

Reply:
yes, it is what the monitoring chart uses https://github.com/rancher/charts/blob/aae93d0125fad561024b25ac1a15580ea612e158/charts/rancher-monitoring/rancher-monitoring/9.4.203/values.yaml#L539-L542

In this case, I can use one of the sub-charts images since I only need to run a bash script in this container.

Reply:
That'd be great, as long as they stay up to date together, ideally using same values.yaml spec